src/sentry/api/urls.py

```diff
@@ -3307,6 +3308,11 @@ def create_group_urls(name_prefix: str) -> list[URLPattern | URLResolver]:
         TestResultsAggregatesEndpoint.as_view(),
         name="sentry-api-0-test-results-aggregates",
     ),
+    re_path(
```

Comment:
i realize this is beyond the scope of this - but these URLs are not maintainable - they should all be prefixed if we're allowing repo access outside of org-scoped queries

e.g. /gh-repos/ or something similar